### PR TITLE
Amount Textfield translations added

### PR DESCRIPTION
**Pull request by edwingl10**
## Motivation and Context

Adds translations to the AmountTextField. 

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] User visible change (users will notice UI or functional changes)

## How Has This Been Tested?

- [ ] Existing or new tests cover my changes.
- [ ] Manually tested locally or on the Render PR Server.

## Previewing Changes

1. Click the `onrender.com` URL the **render `bot`** posted below.
2. add a scholarship
3. change language
4. notice the amountTextField

### CODE REVIEW

**Review by josueetcom — APPROVED** [review]

File: src/components/ScholarshipAmountField.test.jsx [hunk 1]
```diff
@@ -12,13 +14,19 @@ const formik = {
 
 function renderWithAmountType(type) {
   formik.values.amount.type = type;
-  return render(<ScholarshipAmountField formik={formik} />);
+  return render(
+    <Suspense fallback="loading">
+      <I18nextProvider i18n={i18n}>
+        <ScholarshipAmountField formik={formik} />
+      </I18nextProvider>
+    </Suspense>
```
Comment (josueetcom) [inline src/components/ScholarshipAmountField.test.jsx:22]: It seems like we need this just about everywhere. We should consider refactoring out this pattern into our [testing.ts](https://github.com/beyondhb1079/s4us/blob/main/src/lib/testing.ts) library and instead import that `render` function here and everywhere (feel free to do this in a separate PR).

File: src/components/ScholarshipAmountField.test.jsx [hunk 2]
```diff
@@ -12,13 +14,19 @@ const formik = {
...
+  );
 }
 
-test('renders select options', () => {
+test('renders select options', async () => {
   renderWithAmountType(AmountType.Fixed);
 
-  const select = screen.getByRole('button');
+  const select = await waitFor(() => screen.getByRole('button'));
```
Comment (josueetcom) [inline src/components/ScholarshipAmountField.test.jsx:29]: I think you can do this:
```suggestion
  const select = await screen.findByRole('button');
```

Don't know if I forgot to suggest this elsewhere in other PRs.

**Review by josueetcom — APPROVED** [review]

File: src/components/ScholarshipAmountField.test.jsx (hunk 2 above)
Comment (edwingl10) [reply src/components/ScholarshipAmountField.test.jsx:29]: I tried that and it failed the tests because the only things rendered is `loading`

Comment (josueetcom) [reply src/components/ScholarshipAmountField.test.jsx:29]: Weird, `findByRole` _should_ work.

**Pull request merged**